pages/torchbox-academy.tsx

```diff
@@ -13,16 +12,20 @@ type TorchboxAcademyPageProps = {
 const TorchboxAcademyPage: NextPage<TorchboxAcademyPageProps> = ({
     preview,
     content,
-}) => (
-    <Layout theme="INDIGO" preview={preview} jobsAvailable={8}>
-        <h1>Torchbox Academy</h1>
-        <RichText theme="INDIGO" content={content.heroSubtitle} />
-        <Image
-            src={content.heroImage.url}
-            alt={content.heroImage.description}
-        />
-    </Layout>
-);
+}) => {
+    const academyCollectionItem = content.itemsCollection.items.find(
+        (obj: any) => obj.__typename === 'Academies',
+    );
+
+    const academies = academyCollectionItem.academiesCollection.items;
```

Comment:
**Question** Can a editor in contentful leave `itemsCollection` blank? not sure it would hurt to render Academies condtionally.

Reply:
This would be nice to have - I will add a note to double check the nav in the hero doesn't show this section if it is rendering conditionally